src/components/layout/Sidebar.vue

```diff
@@ -5,7 +5,7 @@
         {{ category.categoryName }}
       </p>
       <ul class="menu-list">
-        <li v-for="item in category.components" exact="true" v-bind:key="item.name">
```

Comment:
this one is included in many places actually, I will remove it later everywhere

Reply:
Ah yeah these were the two places where it was constantly complaining, but I did not check for others as they didn't bother me that much 😅